snippets/distr-tracing-tempo-tempostack-custom-resource.adoc

```diff
@@ -0,0 +1,43 @@
+// :_mod-docs-content-type: SNIPPET
+// Text snippet included in the following modules:
+//
+// * modules/distr-tracing-tempo-install-tempostack-web-console.adoc
+// * modules/distr-tracing-tempo-install-tempostack-cli.adoc
+
+[source,yaml]
+----
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+metadata:
+  name: sample
+  namespace: <project_of_tempostack_instance>
+spec:
+  storageSize: <value>Gi # <1>
+  storage:
+    secret: # <2>
+      name: <secret_name> # <3>
+      type: <secret_provider> # <4>
+    tls:
+      enabled: true <5>
+      caName: <ca_certificate_configmap_name> # <6>
+  template:
+    queryFrontend:
+      jaegerQuery:
+        enabled: true
+        ingress:
+          route:
+            termination: edge
+          type: route
+  resources: # <7>
+    total:
+      limits:
+        memory: <value>Gi
+        cpu: <value>m
```

Comment:
Can I include the `resources` section here?

Reply:
yes